src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -28,6 +31,8 @@
 
     private final String documentManagementUrl;
     private final String contextPath;
+    private final ServiceConfigProvider serviceConfigProvider;
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExceptionRecordMapper.class);
```

Comment:
it would be nice to separate this static field from other regular fields. Maybe first line in the class as in all other classes.

Reply:
how about the one below this?
ALLOWED_CLASSIFICATIONS